core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -590,6 +592,22 @@ export class WorkflowEditorComponent implements AfterViewInit, OnDestroy {
           }
         }
       });
+    // fromJointPaperEvent(this.getJointPaper(), "cell:pointerdblclick")
+    // .pipe(untilDestroyed(this))
+    // .subscribe(event => console.log(event));
+
+    const inputPortClickStream = fromJointPaperEvent(this.getJointPaper(), "input-port-label:pointerdown");
+
+    inputPortClickStream
```

Comment:
What is the point of this? If no please remove, also maybe the above stream can also be removed.